docs/installation/configurations.rst

```diff
@@ -0,0 +1,27 @@
+.. _configurations:
+
+=======================
+Configurations
+=======================
+
+Various configurations can be configured within Open Forms to add data and validation to the forms.
+
+BAG configuration
+-----------------
```

Comment:
Please stick to the same type of heading markers as used in the rest of the documentation.

```suggestion
BAG configuration
=================
```